lib/state/index.js

```diff
@@ -11,11 +11,13 @@ import persistState from 'redux-localstorage';
 import appState from '../flux/app-state';
 
 import auth from './auth/reducer';
+import revisions from './revisions/reducer';
 import settings from './settings/reducer';
 
 export const reducers = combineReducers( {
 	appState: appState.reducer.bind( appState ),
 	auth,
+	revisions,
```

Comment:
I'm so happy that you added this in here! the Redux state is a great place for something like it.

one question I do have though that maybe is worth some discussion is _where_ the revision state belongs. is it a property of the app? of a note? of a view?

it appears like we are selecting a "revision" in the sense that we are selecting something more akin to a full note object and not simply a marker like "last tuesday at 8pm."

I'll dig in more later, but I'm pretty sure we don't currently have a list of revisions on the note object itself. that could be really handy to have for times like this probably.